gamepanel/member.php

```diff
@@ -516,7 +516,7 @@ private function muteIcon() {
 	function memberBar()
 	{
 		global $User;
-		if ($this->Game->anon == 'No' || !$this->isNameHidden) 
+		if ($this->Game->anon == 'No' || (!$this->isNameHidden) && isset($this->isNameHidden))

```

Comment:
There has to be a better way to do this - checking to see if it's set seems like we've made a poor architectural choice elsewhere.

either way, the `isset` check should be before `isNameHidden`.

I'll see if I can take a look for a cleaner solution.